File: Dddml.Wms.JavaCommon/src/nodamoney/Money.java
package nodamoney;

import java.math.BigDecimal;

/**
 * Created by Yang on 2016/7/16.
 */
public class Money {

    private BigDecimal amount;

    private String currency;

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }

    public BigDecimal getAmount() {
        return amount;
    }

    public void setAmount(BigDecimal amount) {
        this.amount = amount;
    }

}

File: Dddml.Wms.AdminUI/ts/Money.ts
namespace Dddml.Wms.Domain {
    export class Money {
        private amount:string;
        private currency:string;

        getAmount():string {
            return this.amount;
        }

        setAmount(amount:string) {
            this.amount = amount;
        }

        getCurrency():string {
            return this.currency;
        }

        setCurrency(currency:string) {
            this.currency = currency;
        }
    }
}